src/Sylius/Bundle/ThemeBundle/Tests/Functional/Bundle/TestBundle/Resources/public/theme_asset.txt

```diff
@@ -1 +1 @@
-Vanilla asset
+Theme asset
```

Comment:
These test were passing, "Vanilla asset" is used here to detect issues with theme assets loading. If we're using a theme and "Vanilla asset" shows, then something is wrong with the codebase, not with tests.